### ISSUE TITLE
Warning: No CA bundle found. SSL validation disabled.

### ISSUE THREAD

**Issue by jeroen** [r-windows/checks#79]
May be a problem on winbuilder. E.g. `CytobankAPI` `GoogleKnowledgeGraphR` `VarfromPDB`

**Comment by jeroen** [comment]
Note that curl itself has:

```
  This is libcurl version 7.64.1 with OpenSSL/1.1.1a (Schannel) 
  Using test server: http://hb.opencpu.org 
  -- 1. Failure: Invalid domain raises an error (@test-certificates.R#15)  -------
  `curl_fetch_memory(fake_url)` did not throw an error.
  
  == testthat results  ===========================================================
  [ OK: 216 | SKIPPED: 1 | WARNINGS: 0 | FAILED: 1 ]
  1. Failure: Invalid domain raises an error (@test-certificates.R#15) 
```

**Comment by jeroen** [comment]
Probably the same cause as https://github.com/r-windows/checks/issues/76 (cert bundle is missing).

**Comment by jeroen** [comment]
All fixed via patch suggested in: https://github.com/r-windows/checks/issues/76#issuecomment-556363694
